data/markdown/pages/learn/accelerate/xm-cloud/implementation/search.md

```diff
@@ -1,11 +1,115 @@
 ---
 title: 'Sitecore Accelerate for partners'
-description: 'Learn more about how to develop for XM Cloud'
+description: 'Recipe for implementing search experience with XM Cloud and Search'
 hasSubPageNav: true
 hasInPageNav: false
 area: ['accelerate']
```

Comment:
Front Matter is missing the `lastUpdated` property

Reply:
added